build.sbt

```diff
@@ -55,7 +56,9 @@ ThisBuild / scalafmtPrintDiff := true
 
 // scalafix
 // https://scalacenter.github.io/scalafix/docs/rules/RemoveUnused.html
-ThisBuild / scalacOptions ++= Seq("-Ywarn-unused")
+ThisBuild / scalacOptions ++= Seq("-Ywarn-unused",
+    // Fixes: Exhaustivity analysis reached max recursion depth, not all missing cases are reported.
+    "-Ypatmat-exhaust-depth", "22")
```

Comment:
The comments on 57 and 58 are out of place with the more general use of the compiler options here. Could you move them so they are just over the `-Ywarn-unused` flag now?